web_src/less/_repository.less

```diff
@@ -2571,6 +2571,68 @@ tbody.commit-list {
     padding-bottom: 8px;
 }
 
+.board {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    overflow-x: auto;
+    margin: 0 0.5em;
+}
+
+.board-column {
+    background-color: #eff1f3 !important;
+    border: 1px solid rgba(34, 36, 38, 0.15) !important;
+    margin: 0 0.5em !important;
+    padding: 5px !important;
+    width: 320px;
+    height: 60vh;
+    overflow-y: scroll;
+    flex: 0 0 auto;
+}
+
+.board-column > .cards {
+    margin: 0 !important;
+}
+
+.board-column > .board-label {
+    margin-left: 5px !important;
+    margin-top: 3px !important;
+}
+
+.board-column > .divider {
+    margin: 5px 0;
+}
+
+.board-column:first-child {
+    margin-left: auto !important;
+}
+
+.board-column:last-child {
+    margin-right: auto !important;
+}
+
+.board-card {
+    margin: 3px !important;
+}
+
+.board-card .header {
+    font-size: 1.1em !important;
+}
+
+.board-card .content {
+    padding: 5px 8px !important;
+}
+
+.board-card .extra.content {
+    padding: 5px 8px !important;
+}
+
+.board-label {
+    padding: 0.4em 0.6em !important;
+    margin-right: 0.4em !important;
```

Comment:
```suggestion
    margin-right: .4em !important;
```

Reply:
I think the source should aim for clarity, where as omitting characters should be the job of minifier.